api/src/pcapi/core/finance/api.py

```diff
@@ -88,6 +88,7 @@
 from . import repository
 from . import utils
 from . import validation
+from ...models.feature import FeatureToggle
```

Comment:
```suggestion
from pcapi.models.feature import FeatureToggle
```
On aime pas avoir trop de points :p

Reply:
Bad Pycharm !